src/main/scala/centaur/api/CromwellClient.scala

```diff
@@ -33,6 +35,19 @@ object CromwellClient {
   final implicit val materializer: ActorMaterializer = ActorMaterializer(ActorMaterializerSettings(system))
   final val apiPath = "/api/workflows/v1"
 
+  def decodeResponse(response: HttpResponse): HttpResponse = {
+    val decoder = response.encoding match {
+      case HttpEncodings.gzip ⇒
+        Gzip
+      case HttpEncodings.deflate ⇒
+        Deflate
```

Comment:
oh ok that's even better! So a general purpose client could choose what encoding it wants (but for some reason the akka `Http().singleRequest` is unable to decode what it gets back?)